bin/create-hierarchy

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import argparse
+import pandas as pd
+from pathlib import Path
+
+base = Path(__file__).resolve().parent.parent
+
+import sys
+sys.path.insert(0, str(base / "lib"))
+from utils.hierarchy_dataframe import hierarchy_dataframe
+
+
+HIERARCHY_COLUMNS = ['region', 'country', 'division']
+
+parser = argparse.ArgumentParser(
+    description="Create a known location hierarchy from given metadata that "
+        "can be used to validate future metadata and annotations. The output "
+        "hierarchy TSV is printed to stdout, and you likely want to redirect "
+        "it to a new file.",
+    formatter_class=argparse.RawTextHelpFormatter
+)
+
+parser.add_argument(
+    "metadata",
+    nargs="?",
+    default="s3://nextstrain-ncov-private/metadata.tsv.gz",
+    help="Metadata file (in TSV format) used to build hierarchy.\n")
+
+args = parser.parse_args()
+
+metadata = pd.read_csv(args.metadata, sep='\t')
+
+known_hierarchies = hierarchy_dataframe(metadata, 'gisaid_epi_isl', HIERARCHY_COLUMNS)
```

Comment:
The `gisaid_epi_isl` key may want to be parameterized.

Reply:
I believe it is parametrized?